lib/src/text-input/types.ts

```diff
@@ -60,6 +60,10 @@ type Props = {
    * If true, the component will be disabled.
    */
   disabled?: boolean;
+  /**
+   * If true, the component will not be mutable, meaning the user can not edit the control.
```

Comment:
What do you think about mentioning that it will affect the events' behavior?

Reply:
Agree on that!